Formula/zeromq.rb

```diff
@@ -11,47 +12,56 @@ class Zeromq < Formula
     sha256 "78c24719c7c8dbba157ca225b41d515190281eb27549caaa60a359ec9f33e3f1" => :yosemite
   end
 
-  head do
-    url "https://github.com/zeromq/libzmq.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
   option :universal
   option "with-libpgm", "Build with PGM extension"
   option "with-norm", "Build with NORM extension"
+  option "with-drafts", "Build and install draft classes and methods"
 
   deprecated_option "with-pgm" => "with-libpgm"
 
+  depends_on "asciidoc" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "pkg-config" => :build
+  depends_on "xmlto" => :build
   depends_on "libpgm" => :optional
   depends_on "libsodium" => :optional
+  depends_on "libtool" => :build
   depends_on "norm" => :optional
 
+  # Avoid known, intermittent "make check" failure
+  # upstream issue "make check error: tests/test_shutdown_stress"
+  # https://github.com/zeromq/libzmq/issues/1509
+  # Autotools dependencies can be removed from stable once this is fixed
+  patch :DATA
+
   def install
     ENV.universal_binary if build.universal?
+    ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
 
     args = ["--disable-dependency-tracking", "--prefix=#{prefix}"]
-    if build.with? "libpgm"
-      # Use HB libpgm-5.2 because their internal 5.1 is b0rked.
-      ENV["pgm_CFLAGS"] = `pkg-config --cflags openpgm-5.2`.chomp
-      ENV["pgm_LIBS"] = `pkg-config --libs openpgm-5.2`.chomp
-      args << "--with-pgm"
-    end
 
-    if build.with? "libsodium"
-      args << "--with-libsodium"
+    args << "--with-pgm" if build.with? "libpgm"
+    args << "--with-libsodium" if build.with? "libsodium"
+    args << "--with-norm" if build.with? "norm"
+
+    if build.with?("drafts")
+      args << "--enable-drafts"
     else
-      args << "--without-libsodium"
+      args << "--disable-drafts"
     end
 
-    args << "--with-norm" if build.with? "norm"
+    system "./autogen.sh"
+
+    ENV["LIBUNWIND_LIBS"] = "-framework System"
+    ENV["LIBUNWIND_CFLAGS"] = "-I#{MacOS.sdk_path}/usr/include"
 
-    system "./autogen.sh" if build.head?
     system "./configure", *args
     system "make"
+    system "make", "check", "TESTS=''"
+    ENV.deparallelize do
+      system "sh", "-c", "ulimit -n 1200; make check"
```

Comment:
Bit concerned that we need this and the previous call and the patch. Is it not just worth removing `make check` for now?